### PR TITLE
feat: add custom extract variable text per language for 119 operation in global config

### PR DESCRIPTION
**Pull request by teddylear**
feat: add custom extract variable text per language for 119 operation in global config

Solves part of #350, but I want to make a second PR for per call config for customization like we have for debugging.

### CODE REVIEW

File: lua/refactoring/config/init.lua [hunk 1]
```diff
@@ -71,6 +71,7 @@ function Config:new(...)
         prompt_func_param_type = default_prompt_func_param_type,
         printf_statements = default_printf_statements,
         print_var_statements = default_print_var_statements,
+        extract_new_var_statements = default_extract_new_var_statements,
```
Comment (pranavrao145) [inline lua/refactoring/config/init.lua:74]: Should we perhaps call this `extract_var_statements`, just to be consistent with the other settings (each one references the name of the operation they're associated with)?

**Review by pranavrao145 — APPROVED** [review]
Left a comment, take a look and let me know what you think - other than that, looks good to me; good to merge if you are!

File: lua/refactoring/config/init.lua (hunk 1 above)
Comment (teddylear) [reply lua/refactoring/config/init.lua:74]: Sure let me update.

Comment (teddylear) [reply lua/refactoring/config/init.lua]: Updated for new name, I'll merge.

**Pull request merged**